todoApp/spec/tspconfig.yaml

```diff
@@ -18,4 +19,10 @@ options:
   "@typespec/http-client-java":
     emitter-output-dir: "{project-root}/../clients/java"
   "@typespec/http-client-python":
-    emitter-output-dir: "{project-root}/../clients/python"
\ No newline at end of file
+    emitter-output-dir: "{project-root}/../clients/python"
+  "@azure-tools/typespec-ts":
+    isModularLibrary: true
+    azureSdkForJs: false
```

Comment:
Can we let the default to be false to hide this config:  `azureSdkForJs: false`

Reply:
I think we can remove this since https://github.com/Azure/autorest.typescript/issues/2329 has been resolved. @v-jiaodi  could you have a try to remove it? if you see anything is changing after removing this option, could you work with @kazrael2119 for it? thanks